languages/julia/concepts/iterator-protocol/links.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "url": "https://invenia.github.io/blog/2020/11/06/interfacetesting/",
+        "description": "Development with Interface Packages",
+        "__why": "Julia's notion of informal interfaces is different to interfaces in many other languages. More experienced students may wonder how one can reliably define and test such interfaces. This post outlines how Invenia, and several of their open-sourced packages, do it in a production environment."
```

Comment:
This is a good blog post, but without the context of the comment, I think students will think they have to understand all this to use iterators, which obviously they do not.

I think if this is linked it should be linked from some prose with context.

Reply:
Makes sense. I'll move it to the about once it's written. Perhaps "informal interfaces" should even be its own concept?

Reply:
Perhaps. If so, this exercise already teaches it a little. Add it to the pile of maybes? :)

Reply:
Sounds good. I removed the link for now and will create follow-up issues later.

Reply:
See #2834